src/Point.ts

```diff
@@ -384,5 +385,3 @@ export class Point implements IPoint {
     );
   }
 }
-
-const originZero = new Point(0, 0);
```

Comment:
this seems unused and with sligthly less minification potential, why the change?

Reply:
It was used before I refactored the logic
That I thought it makes sense and left it. I can dump it.